pkg/infrastructure/platform/platform_altinfra.go

```diff
@@ -22,6 +23,11 @@ func ProviderForPlatform(platform string, fg featuregates.FeatureGate) (infrastr
 	case azuretypes.Name:
 		panic("not implemented")
 		return nil, nil
+	case gcptypes.Name:
+		if fg.Enabled(configv1.FeatureGateClusterAPIInstall) {
```

Comment:
It looks like a few imports are missing which is causing the altinfra-images test to fail
pkg/infrastructure/platform/platform_altinfra.go:27:17: undefined: configv1
pkg/infrastructure/platform/platform_altinfra.go:28:11: undefined: clusterapi
pkg/infrastructure/platform/platform_altinfra.go:28:41: undefined: gcpcapi